modules/asr/Cyberon/flash_read_ext.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021-2022 XMOS LIMITED.
```

Comment:
date

Reply:
Most of the files in the repo have the wrong date, I will update them after this PR is merged.

Reply:
Just a comment - the dates could be correct if they were the last modification date of the specific files.